host_core/lib/host_core/web_assembly/imports.ex

```diff
@@ -468,12 +466,10 @@ defmodule HostCore.WebAssembly.Imports do
   end
 
   defp guest_request(_api_type, context, agent, op_ptr, ptr) do
-    Logger.debug("Guest request")
```

Comment:
It's logs like this that make me wish Logger had a trace that we could be this verbose in, but I digress